lib/condor.py

```diff
@@ -93,44 +94,52 @@ def load_submit_file(filename):
     return htcondor.Submit(res), nqueue
 
 
-def submit(f, vargs, schedd_add, cmd_args=[]):
+def submit(f, vargs, schedd_args, schedd_name, cmd_args=[]):
```

Comment:
As far as I can see, we _only_ use `schedd_name` for the setting of `_condor_CREDD_HOST` here.  From above, it seems like both in `jobsub_submit` and `condor_submit`, the schedd is already set in the `schedd_args` (always after `-name` or `-remote`).  This seems like a redundancy.

Is there a good way to pull the schedd out of the `schedd_args` so the function signature doesn't need `schedd_name`?